scripts/update_script.sh

```diff
@@ -1,19 +1,33 @@
 #!/bin/bash
 # This script performs:
-# pull origin main
+# fetch origin main
 #- if any change:
-#    kill container
-#    rebuild docker compose
+#    pull changes
+#    any others change needed
 # 
-# This script must be run with a crontab, run every day at 3am
-# 0 3 * * * bash /home/pi/pyro-engine/scripts/update_script.sh
+# This script must be run with a crontab, run every hour
+# 0 * * * * bash /home/pi/pyro-engine/scripts/update_and_execute.sh >> /home/pi/pyro-engine/logfile.log 2>&1
```

Comment:
the name of the current file does not seem to have changed so it might be 
```suggestion
# 0 * * * * bash /home/pi/pyro-engine/scripts/update_script.sh >> /home/pi/pyro-engine/logfile.log 2>&1
```
so this command for a CRON is set up at setup time, I am right ?

Reply:
oh yes thansk